README.md

```diff
@@ -1013,6 +1014,12 @@ roborazzi {
 }
 ```
 
+> **Note**  
+> If you are using build.gradle instead of build.gradle.kts, you need to use the set method for each assignment, like
+> ```kotlin
+> generateComposePreviewRobolectricTests.packages.set(["com.example"])
+> ```
+
```

Comment:
In README.md, highlighting using the [!NOTE] syntax did not work.
so, I used **Note**, which is the most natural looking.  

- screenshot using [!NOTE] syntax  in actual README.md
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/0f1215aa-32ec-41b3-9391-ebf841b865a4">